documentation/book/proc-restricting-access-to-listeners-using-network-policies.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// assembly-configuring-kafka-listeners.adoc
+
+[id='proc-restricting-access-to-listeners-using-network-policies-{context}']
+= Restricting access to Kafka listeners using NetworkPolicies
+
+.Prerequisites
+
+* An {ProductPlatformName} cluster with SDN supporting Ingress NetworkPolicies
```

Comment:
SDN > software-defined networking (SDN)
supporting > that supports

(Include full stop)